src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="content">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```

Comment:
- double-dash one (`--`) refers to modifiers, single-dash one (`-`) – just a part of a name, not BEM-related, consider it when giving names
- let's format it properly
```suggestion
    <div 
      class="
        calendar 
        calendar--start-day-sun 
        calendar--month-length-29
      "
    >
```

Reply:
done